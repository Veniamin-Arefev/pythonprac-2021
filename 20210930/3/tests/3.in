67, -96, 76, -8, -24
33, -63, 29, 24, 19
95, -11, 90, 73, 88
89, 21, 85, 30, 71
78, -99, -52, -87, 92
-45, -68, -1, 19, -92
-53, -11, -10, 63, 4
-22, 0, 99, 37, -60
70, -1, 38, -92, -46
54, 5, -5, 17, 33