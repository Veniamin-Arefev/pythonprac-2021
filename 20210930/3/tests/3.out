[-1455, -3612, 8233, -1635, -11532]
[3922, -1480, 4285, -4154, -5505]
[4190, -5972, 11259, -778, -14638]
[-1054, -5958, 8901, 4606, -12241]
[1759, -3668, -8002, 2889, 2586]
